Source/DafnyCore/AST/Grammar/Printer.cs

```diff
@@ -408,7 +408,10 @@ public class Printer {
           wr.WriteLine();
           Indent(indent);
           if (d is LiteralModuleDecl modDecl) {
-            if (printMode == PrintModes.DllEmbed && !modDecl.ModuleDef.IsToBeCompiled) {
+            if (printMode == PrintModes.Serialization && !modDecl.ModuleDef.IsToBeCompiled) {
```

Comment:
Does this mean that we wouldn't support proof-only libraries?

Reply:
Oh sure we will - the `IsToBeCompiled` field really just means "was not provided by `--library`". So you can work just with `.doo` files and never compile any of them with the other backends. Perhaps the name could be "IsToBeBuilt" instead, but I don't think that's enough of an improvement to bother with personally. :)

Reply:
Ah, right. Sounds good!